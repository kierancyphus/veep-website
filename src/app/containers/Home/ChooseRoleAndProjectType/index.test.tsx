import React from 'react'
import {
  ChooseRoleAndProjectType,
  ChooseRoleAndProjectTypeProps,
} from './index'
import { render } from '@testing-library/react'
import { BrowserRouter } from 'react-router-dom'

describe('ChooseRoleAndProjectType', () => {
  const props: ChooseRoleAndProjectTypeProps = {
    userType: 0,
    projectType: 0,
    userTypeToStringMap: { 0: 'user' },
    projectTypeToStringMap: { 0: 'project' },
    setProjectType: jest.fn(),
    setUserType: jest.fn(),
  }

  it('renders without errors', () => {
    const { container: firstChild } = render(
      <ChooseRoleAndProjectType {...props} />,
      { wrapper: BrowserRouter },
    )

    expect(firstChild).toMatchSnapshot()
  })
})
