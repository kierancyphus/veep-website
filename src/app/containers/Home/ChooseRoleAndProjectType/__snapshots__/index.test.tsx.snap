// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ChooseRoleAndProjectType renders without errors 1`] = `
<div>
  <div
    class="MuiBox-root MuiBox-root-2"
  >
    <div
      class="react-reveal"
    >
      <div
        class="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-1"
      >
        <div
          class="MuiGrid-root MuiGrid-item MuiGrid-grid-sm-12 MuiGrid-grid-lg-7"
        >
          <div
            class="MuiBox-root MuiBox-root-4"
          >
            <div
              class="MuiBox-root MuiBox-root-6"
            >
              <h6
                class="MuiTypography-root MuiTypography-h6"
              >
                Role
              </h6>
            </div>
            <div
              class="MuiBox-root MuiBox-root-8"
            >
              <div
                class="MuiBox-root MuiBox-root-10"
              >
                <button
                  class="MuiButtonBase-root MuiButton-root MuiButton-contained"
                  tabindex="0"
                  type="button"
                >
                  <span
                    class="MuiButton-label"
                  >
                    user
                  </span>
                  <span
                    class="MuiTouchRipple-root"
                  />
                </button>
              </div>
            </div>
          </div>
          <div
            class="MuiBox-root MuiBox-root-12"
          >
            <div
              class="MuiBox-root MuiBox-root-14"
            >
              <h6
                class="MuiTypography-root MuiTypography-h6"
              >
                Project Type
              </h6>
            </div>
            <div
              class="MuiBox-root MuiBox-root-16"
            >
              <div
                class="MuiBox-root MuiBox-root-18"
              >
                <button
                  class="MuiButtonBase-root MuiButton-root MuiButton-contained"
                  tabindex="0"
                  type="button"
                >
                  <span
                    class="MuiButton-label"
                  >
                    project
                  </span>
                  <span
                    class="MuiTouchRipple-root"
                  />
                </button>
              </div>
            </div>
          </div>
        </div>
        <div
          class="MuiGrid-root MuiGrid-item MuiGrid-grid-lg-5"
        >
          <div
            class="MuiBox-root MuiBox-root-20"
          >
            This is where all the "you are a ..." and "through this project you will develop..." To save on space we could use Icons for a lot of the technical skills
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
